webknossos/webknossos/client/README.md

```diff
@@ -0,0 +1,28 @@
+# webKnossos REST-API client
+
+:warning: The code of this client is auto-generated! Please don't adapt manually, but re-run the code generation. :warning:
+
+The client code is generated using [openapi-python-client](https://github.com/openapi-generators/openapi-python-client) (version 0.10.3, Python 3.7.3).
+
+To re-generate,
+1. [install openapi-python-client](https://github.com/openapi-generators/openapi-python-client#installation),
+2. move into the `client` directory (`cd webknossos/webknossos/client`),
+3. run `openapi-python-client generate --url https://raw.githubusercontent.com/openapi-generators/openapi-python-client/main/end_to_end_tests/openapi.json`,
+4. move the relevant files into the correct folder:
+   ```
+   mv webknossos-client/webknossos_client/api .
+   mv webknossos-client/webknossos_client/models .
+   mv webknossos-client/webknossos_client/*.py .
+   ```
```

Comment:
Is it correct that there is `webknossos/webknossos/client`, `webknossos-client/webknossos_client` and `my-test-api-client`? I have to admit that I'm a bit lost with the instructions here. Maybe it's clearer when once I followed it step-by-step, so, not sure whether this needs to change.

Reply:
Sorry, I just temporarily copied the `webknossos/webknossos/client/README.md`, I removed this now. I'll double-check this anyways when we actually introduce the client, so I guess the current version should be fine.